### PR TITLE
MySQL config guide

### PR DESCRIPTION
**Pull request by oatkachenko**
- How to enable UTF8MB4 character set by default.

### CODE REVIEW

File: docs/installation/general-installation-guide.md [hunk 1]
```diff
@@ -23,3 +23,22 @@ So if you want to run the app locally (from binaries or sources), you need to in
  - Clone the project: `git@github.com:jtalks-org/jcommune.git`. Now you can work with that project from your IDE.
  - If you want to deploy it from command line.. Step into the folder and build it: `mvn clean package`
  - Repeat everything from _Installing from Binaries_ but instead of downloading war-file, just grab it from `jcommune/jcommune-view/jcommunew-web-view/target`
+
+#### Advanced configuring MySQL Server
```
Comment (ctapobep) [inline docs/installation/general-installation-guide.md]: Advanced configuration of MySQL Server

File: docs/installation/windows/basic-environment.md [hunk 2]
```diff
@@ -8,6 +8,8 @@ MySQL:
  - [Install latest version](http://dev.mysql.com/downloads/mysql)
  - Log into MySQL terminal, it may look like `mysql -uroot -proot`
  - Create database: `create database jtalks character set utf8`
```
Comment (ctapobep) [inline docs/installation/windows/basic-environment.md]: utf8?

File: docs/installation/windows/basic-environment.md [hunk 3]
```diff
@@ -8,6 +8,8 @@ MySQL:
...
+ - Note that if you want to use full functionality of jcommune, you need to do a few simple steps described in `\jcommune\docs\installation\general-installation-guide.md`
```
Comment (ctapobep) [inline docs/installation/windows/basic-environment.md]: - NB: some functionality (like posting extended unicode symbols e.g. smiles) is not going to work with default MySQL configuration, read details in the [Advanced configuration of MySQL](link here)

**Comment by oatkachenko** [comment]
@ctapobep fixed!

**Pull request merged**